test/integration/extensions/test_compression_plugin.py

```diff
@@ -0,0 +1,106 @@
+import os
+import textwrap
+
+from conan.test.assets.genconanfile import GenConanfile
+from conan.test.utils.tools import TestClient
+
+
+def test_compression_plugin_not_existing():
+    """Test that the compression plugin is not used if it does not exist"""
+    c = TestClient()
+    c.save({"conanfile.py": GenConanfile("pkg", "1.0")})
+    c.run("create .")
+    c.run("cache save 'pkg/*'")
+    assert "Compressing conan_cache_save.tgz\n" in c.out
+    c.run("cache restore conan_cache_save.tgz")
+    # Default decompress does not have any output
+    assert "Decompressing conan_cache_save.tgz" not in c.out
```

Comment:
Isn't this the standard behavior of the existing ``cache save/restore`` tests? What is this test checking that is not covered by those existing tests? Maybe it is just enough to add this last assert in one existing test?

Reply:
Yes, that is true, this test was made to simply check that if there is no present plugin, the decompression and compression will use default ones. I'll move last asserts to existing tests.